pkg/serverless/protocol.go

```diff
@@ -295,35 +297,14 @@ func (l *LogsCollection) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(400)
 	} else {
 		metricsChan := l.daemon.aggregator.GetBufferedMetricsWithTsChannel()
-		metricTags := getTagsForEnhancedMetrics()
 		sendLogsToIntake := config.Datadog.GetBool("serverless.logs_enabled")
+		computeEnhancedMetrics := config.Datadog.GetBool("enhanced_metrics")
```

Comment:
I would rename `computeEnhancedMetrics` to `enhancedMetricsEnabled` and rename `sendLogsToIntake` to `logsEnabled` -- to make it more clear that they are not functions but booleans.

Reply:
✅